lib/graphql/interface_type.rb

```diff
@@ -64,5 +64,23 @@ def get_field(field_name)
     def all_fields
       fields.values
     end
+
+    # Get a possible type of this {InterfaceType} by type name
+    # @param type_name [String]
+    # @param ctx [GraphQL::Query::Context] The context for the current query
+    # @return [GraphQL::ObjectType, nil] The type named `type_name` if it exists and implements this {InterfaceType}, (else `nil`)
+    def get_possible_type(type_name, ctx)
```

Comment:
I just duplicated these methods for both types right now even though they are identical (other than doc strings). Would you rather these be moved into a module or something?